translator/vector/symbol/utils.py

```diff
@@ -19,3 +19,15 @@ def to_rgba(color: QColor):
     alpha = hex(int(color.alphaF() * 255))[2:].zfill(2)
 
     return f"{color.name()}{alpha}"
+
+def get_stroke_width_pt(defined_width: float, unit: QgsUnitTypes.RenderUnit) -> float:
+    """ get line width in points, considering hairline or not 
+    symbol layer stroke width is set as 0 when hairline
+    to be render as 0.5px (0.375pt) as defined in QGIS code
+    https://github.com/qgis/QGIS/blob/e7f5b6f286b6cbaeaad5e73d69fad1448018c7e0/
+    tests/src/python/test_qgssymbollayer_createsld.py#L234
+    """
+    if defined_width == 0:
+        return 0.375 # pt (hairline size: 0.5px)
```

Comment:
nits: 
It may be better to take more generic approach, what do you think?
Does it bring more complexity without reasonable benefits?

```suggestion
        hairline_width = 0.5
        return convert_to_point(hairline_width, PIXEL) # PIXEL should be replaced with correct one
```

Reply:
@Kanahiro 
I tried with logic implemented in convert_to_point() function, but it is not showed expected result 0.375pt [as expected](https://convertermaniacs.com/pixel-to-point/convert-0.5-px-to-pt.html).

Should I follow QGIS way of conversion from pixel to pt even if is different from common ways (1px = 0.75pt) ?.
![image](https://github.com/MIERUNE/qgis-plugx-plugin/assets/26103833/58dd120c-f77c-4b3b-866b-75e9e8bc1f48)

Reply:
IMHO, pixels depends on DPI so it might not fixed value...
Hairline menas the thinest width then it is not bad to use fixed value, it may be okay that it is not 0.375pt.

Reply:
@Kanahiro Okay method changed